erlang/src/guestbook_images_handler.erl

```diff
@@ -23,11 +24,12 @@ rest_init(Req, Opts) ->
   {ok, Req, #{craterl => ClientRef}}.
 
 allowed_methods(Req, State) ->
-  {[<<"GET">>, <<"POST">>], Req, State}.
+  Req2 = addCORSHeaders(Req),
```

Comment:
if you want to add the CORS headers for every request for this handler, i would suggest adding them in `init`.

Adding them here works, but only because in erlang all variables are assign-once only and the cowboy engine needs to use the request returned here. This method should actually not modify the request, that's why i'd rather put the `addCORSHeaders` call in `init`.

same for all other handlers.